packages/alexa-types/src/version/index.ts

```diff
@@ -24,7 +26,8 @@ export interface PlatformData extends VoiceVersion.PlatformData<Voice> {
   publishing: Publishing;
 }
 
-export interface Version extends VoiceVersion.Version<Voice, BaseModels.Version.Prototype<AnyCommand, VoiceflowConstants.Locale>> {
+export interface Version
+  extends VoiceVersion.Version<Voice, BaseModels.Version.Prototype<AnyCommand, VoiceflowConstants.Locale, AlexaSurveyContextExtension>> {
```

Comment:
Here, I'm just passing in the `AlexaSurveyContextExtension` to extend the `SurveyContext` with Alexa-specific properties for this default "AlexaVersion" exported by `alexa-types`.

In the code below, you'll see me do this a lot of places to avoid type issues in the `*-service`s.